src/ecdsa/test_ellipticcurve.py

```diff
@@ -126,3 +126,34 @@ def test_multiply(c, x1, y1, m, x3, y3):
     ids=["g_23 test with mult {0}".format(i) for i in range(9)])
 def test_add_and_mult_equivalence(p, m, check):
     assert p * m == check
+    
+    
+class TestCurve(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.c_23 = CurveFp(23, 1, 1)
+        
+    def test_equality_curves(self):
+        self.assertEqual(self.c_23, CurveFp(23, 1, 1))
+        
+    def test_inequality_curves(self):
+        c192 = CurveFp(p, -3, b)
```

Comment:
the other test cases use global variables because they use the functional style and the pytest framework, this one uses the object-based approach from unittest, so I'd say that the use of global variables is less clean here...

Reply:
Moved to classes. Not sure if it is possible to convert test_p192_mult_tests preserving functionality.

Reply:
I mean, I'm fine with either approach, what I'm not a fan of is mixing them